### PR TITLE
EDIT: CA318 - facebook URLs and syntax

### PR DESCRIPTION
**Pull request by dominicusplatus**

### CODE REVIEW

File: CollAction/Controllers/AdminController.cs [hunk 1]
```diff
@@ -232,7 +232,7 @@ public IActionResult Index()
                         "<br>" +
                         "The CollAction Team has reviewed your project proposal and is very happy to share that your project has been approved and now live on www.collaction.org!<br>" +
                         "<br>" +
-                        "So feel very welcome to start promoting it! If you have any further questions, feel free to contact the CollAction Team at collactionteam@gmail.com. And don’t forget to tag CollAction in your messages on social media so we can help you spread the word(FB: @collaction.org, Twitter: @collaction_org)!<br>" +
+                        "So feel very welcome to start promoting it! If you have any further questions, feel free to contact the CollAction Team at collactionteam@gmail.com. And don’t forget to tag CollAction in your messages on social media so we can help you spread the word ( FB: @<a href='https://www.facebook.com/collaction.org/'>collaction.org</a>, Twitter: @collaction_org)!<br>" +
```
Comment (ljwboschker) [inline CollAction/Controllers/AdminController.cs]: There is an extra space after the (

File: CollAction/Controllers/AdminController.cs [hunk 2]
```diff
@@ -250,7 +250,7 @@ public IActionResult Index()
                         "<br>" +
                         "The deadline of the project you have started on www.collaction.org has passed. We're very happy to see that the target you have set has been reached! Congratulations! Now it's time to act collectively!<br>" +
                         "<br>" +
-                        "The CollAction Team might reach out to you with more specifics (this is an automated message). If you have any further questions yourself, feel free to contact the CollAction Team at collactionteam@gmail.com. And don’t forget to tag CollAction in your messages on social media so we can help you spread the word on your achievement (FB: @collaction.org, Twitter: @collaction_org)!<br>" +
+                        "The CollAction Team might reach out to you with more specifics (this is an automated message). If you have any further questions yourself, feel free to contact the CollAction Team at collactionteam@gmail.com. And don’t forget to tag CollAction in your messages on social media so we can help you spread the word on your achievement ( FB: @<a href='https://www.facebook.com/collaction.org/'>collaction.org</a>, Twitter: @collaction_org)!<br>" +
```
Comment (ljwboschker) [inline CollAction/Controllers/AdminController.cs]: Also an extra space

File: CollAction/Controllers/AdminController.cs (hunk 1 above)
Comment (logophobia) [reply CollAction/Controllers/AdminController.cs]: Don't we want the "@" inside the link?

File: CollAction/Controllers/AdminController.cs (hunk 2 above)
Comment (logophobia) [reply CollAction/Controllers/AdminController.cs]: Don't we want the "@" inside the link?

Comment (dominicusplatus) [reply CollAction/Controllers/AdminController.cs]: @ljwboschker this was requested, CA-318 :  
" 1. "word(FB: @collaction.org," --> add a space between "word" and the "(" "
@logophobia  this is now fixed

File: CollAction/Controllers/AdminController.cs [hunk 3]
```diff
@@ -232,7 +232,7 @@ public IActionResult Index()
                         "<br>" +
                         "The CollAction Team has reviewed your project proposal and is very happy to share that your project has been approved and now live on www.collaction.org!<br>" +
                         "<br>" +
-                        "So feel very welcome to start promoting it! If you have any further questions, feel free to contact the CollAction Team at collactionteam@gmail.com. And don’t forget to tag CollAction in your messages on social media so we can help you spread the word(FB: @collaction.org, Twitter: @collaction_org)!<br>" +
+                        "So feel very welcome to start promoting it! If you have any further questions, feel free to contact the CollAction Team at collactionteam@gmail.com. And don’t forget to tag CollAction in your messages on social media so we can help you spread the word ( FB: <a href='https://www.facebook.com/collaction.org/'>@collaction.org</a>, Twitter: @collaction_org)!<br>" +
```
Comment (logophobia) [inline CollAction/Controllers/AdminController.cs]: I think you added one space too much (looking at CA-318 on JIRA).

There's a space added between FB and (: "( FB" that probably shouldn't be there.

File: CollAction/Controllers/AdminController.cs [hunk 4]
```diff
@@ -250,7 +250,7 @@ public IActionResult Index()
                         "<br>" +
                         "The deadline of the project you have started on www.collaction.org has passed. We're very happy to see that the target you have set has been reached! Congratulations! Now it's time to act collectively!<br>" +
                         "<br>" +
-                        "The CollAction Team might reach out to you with more specifics (this is an automated message). If you have any further questions yourself, feel free to contact the CollAction Team at collactionteam@gmail.com. And don’t forget to tag CollAction in your messages on social media so we can help you spread the word on your achievement (FB: @collaction.org, Twitter: @collaction_org)!<br>" +
+                        "The CollAction Team might reach out to you with more specifics (this is an automated message). If you have any further questions yourself, feel free to contact the CollAction Team at collactionteam@gmail.com. And don’t forget to tag CollAction in your messages on social media so we can help you spread the word on your achievement ( FB: <a href='https://www.facebook.com/collaction.org/'>@collaction.org</a>, Twitter: @collaction_org)!<br>" +
```
Comment (logophobia) [inline CollAction/Controllers/AdminController.cs]: I think you added one space too much (looking at CA-318 on JIRA).

There's a space added between FB and (: "( FB" that probably shouldn't be there.

Comment (dominicusplatus) [reply CollAction/Controllers/AdminController.cs]: This is fixed in 8458770

File: CollAction/Controllers/AdminController.cs (hunk 3 above)
Comment (dominicusplatus) [reply CollAction/Controllers/AdminController.cs]: This is fixed in 8458770

**Pull request merged**